.github/workflows/CI.yml

```diff
@@ -62,6 +62,11 @@ jobs:
             CROSS_ARCH: arm64
             CROSS_PACKAGES: libgtk-3-dev:arm64 libatk-bridge2.0-dev:arm64 libgdk-pixbuf2.0-dev:arm64 libglu-dev:arm64 libgl1-mesa-glx:arm64 libx11-dev:arm64
             NATIVE_PARAMS: -Dgcc.libpath.opengl=/usr/lib/aarch64-linux-gnu/mesa
+          - ARCH: ppc64el
+            PACKAGES: gcc-powerpc64le-linux-gnu g++-powerpc64le-linux-gnu gcc-powerpc64le-linux-gnu g++-powerpc64le-linux-gnu libc6-dev-ppc64el-cross
+            CROSS_ARCH: ppc64el
+            CROSS_PACKAGES: libgtk-3-dev:ppc64el libatk-bridge2.0-dev:ppc64el libgdk-pixbuf2.0-dev:ppc64el libglu-dev:ppc64el libgl1-mesa-glx:ppc64el libx11-dev:ppc64el
+            NATIVE_PARAMS: -Dgcc.libpath.opengl=/usr/lib/powerpc64le-linux-gnu/mesa -Dbinding.sse=false -Dbinding.meow=false       
```

Comment:
NOTE:

The `binding.sse` is set to `false` because PowerPC ISA does not have SSE
The  `binding.meow` is set to `false` because there is no support for crypto at this stage yet. There is AES support in PowerISA but AFAIK it is not yet implemented here